cves/CVE-2014-8547.yml

```diff
@@ -90,18 +90,17 @@ fixes_vcc_instructions: |
   find interesting in there.
 fixes:
 - commit: 8f1457864be8fb9653643519dea1c6492f1dde57
-  note: ''
-- commit: ''
-  note: ''
+  note: |
```

Comment:
We don't need the full commit message here (we could have grabbed that automatically). You can leave it blank, or add any pertinent information that's not obvious from the commit message or looking at the fix code.